.travis.yml

```diff
@@ -17,32 +17,39 @@ cache:
     - $HOME/.android/build-cache
 android:
   components:
-    - tools
+    - tools # to get the new `repository-11.xml`
+    - tools # see https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943)
```

Comment:
Does this need to be duplicated? I see the `not a mistake` comment below.

Reply:
Yes, travis has this issue for a while now